Makefile

```diff
@@ -189,8 +189,17 @@ helm-install-druid-operator: ## helm upgrade/install
 	helm upgrade --install \
 	--namespace ${NAMESPACE_DRUID_OPERATOR} \
 	--create-namespace \
+	zookeeper bitnami/zookeeper
```

Comment:
ZooKeeper should be deployed on another namespace - not in the druid operator namespace